src/Tizen.Multimedia.MediaPlayer/Player/AdaptiveVariants.cs

```diff
@@ -0,0 +1,179 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Diagnostics;
+using System.Collections.Generic;
+using static Interop;
+
+namespace Tizen.Multimedia
+{
+    /// <summary>
+    /// Represents properties for the variant information
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
+    public struct VariantInfo
+    {
+        /// <summary>
+        /// Initializes a new instance of the VariantInfo struct.
+        /// </summary>
+        /// <param name="bandwidth">The bandwidth of the stream can be supportable, it must be set. (deafult: -1)</param>
+        /// <param name="width">The width of the stream, this is optional parameter. (deafult: -1)</param>
+        /// <param name="height">The height of the stream, this is optional parameter. (deafult: -1)</param>
+        /// <since_tizen> 5 </since_tizen>
+        public VariantInfo(int bandwidth, int width, int height)
```

Comment:
If width and height is optional value, how about use defalut parameter?

Reply:
that's a good idea. i'll fix it.